network/network.go

```diff
@@ -227,7 +227,7 @@ func (n *Network) Start() error {
 	n.startTime.Store(time.Now())
 
 	// Load DAG and start publishing
-	n.publisher.Subscribe(dag.AnyPayloadType, n.lastTransactionTracker.process)
+	n.publisher.Subscribe(dag.TransactionPayloadAddedEvent, dag.AnyPayloadType, n.lastTransactionTracker.process)
```

Comment:
should be TransactionAddedEvent

Reply:
As long as all previous payloads must be present in all cases `prevs` MUST refer to TXs which payload is guaranteed to be present. Otherwise the publisher will not directly publish (or maybe never, if the payload is forever lost) the TX (added by the local node through `CreateTransaction`). This is what `lastTransactionTracker` tracks.

Ofc might change in future or be phased out completely when TXs are guaranteed to have their payloads, due to atomic sync (where applicable)

Reply:
As long as all previous payloads must be present in all cases `prevs` MUST refer to TXs which payload is guaranteed to be present. Otherwise the publisher will not directly publish (or maybe never, if the payload is forever lost) the TX (added by the local node through `CreateTransaction`). This is what `lastTransactionTracker` tracks. Thus, it needs to listen to the `TransactionPayloadEvent`.

Ofc might change in future or be phased out completely when TXs are guaranteed to have their payloads, due to atomic sync (where applicable)

Reply:
So private TX will only be referred to by other TX of the same node, which is acceptable for now.